docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,109 @@
+
+import { AndroidIcon } from '@expo/styleguide-icons/custom/AndroidIcon';
+import { AppleIcon } from '@expo/styleguide-icons/custom/AppleIcon';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+
+## Linking
+
+Linking allows your app to interact with incoming and outgoing URLs. In this process, the user not only gets directed to open your app, but they are taken to a specific screen (route) within the app.
+
+### Linking strategies
+
+There are different linking strategies you handle in your Expo app:
+
+- Linking to other apps from your app (outgoing links)
+- Linking to your app from other apps or a web browser (incoming links)
+- Linking to specific content within your app (deep links)
+- Redirecting links to your web domain to your app if it is installed (Android App Links and iOS Universal Links); also known as App shortcuts
+
+> **info** **Tip:** We recommend using [Development builds](/develop/development-builds/introduction/) to test your app's linking strategies instead of Expo Go.
+
+## Linking to other apps from your app
+
+Linking to other apps from your app is achieved using a URL based on the target app's URL scheme. This **URL scheme** allows you to reference resources within that native app.
+
+Your app can use a [common URL scheme](/linking/into-other-apps/#common-url-schemes) for default apps or implement any URL scheme, including `https` and `http` (commonly used by web browsers like Chrome, Safari, and so on), and use JavaScript to handle the URL that launches the corresponding native app.
+
+<BoxLink
+  title="Linking into other apps"
+  description="Learn how to handle common and custom URL schemes to link other apps from your app."
+  href="/linking/into-other-apps/"
+  Icon={BookOpen02Icon}
+/>
+
+## Linking to your app from other apps or a web browser
+
+[Deep Links](https://en.wikipedia.org/wiki/Deep_linking) are links to a specific URL based content inside an app or a website.
+
+For example, by clicking an advertisement of your product, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like:
+
+```html
+myapp://web-app.com/product
+```
+
+This link is constructed by three parts:
+
+- **Scheme**: The URL scheme that identifies the app that should open the URL (example: `myapp://`). It can also be `https` or `http` for non-standard deep links.
+- **Host**: The domain name of the app that should open the URL (example: `web-app.com`).
+- **Path**: The path to the screen that should be opened (example: `/product`). If the path isn't specified, the user is taken to the home screen of the app.
+
+<BoxLink
+  title="Linking to your app"
+  description="Learn how to configure custom URL schemes to create a deep link of your app."
+  href="/linking/into-your-app/"
+  Icon={BookOpen02Icon}
+/>
+
+## App shortcuts
```

Comment:
```suggestion
## Universal linking

Both Android and iOS implement their own systems for routing web URL's to an app if the app is installed. On Android, this system is called App Links, and on iOS it is called Universal Links.
```
Maybe something like this? I do tend to see this referred to as "universal links" or "universal linking" colloquially.

Reply:
Yeah, we used to call the group "universal links" before. Sounds good to me!